src/main/scala/edu/ie3/simona/model/thermal/CylindricalThermalStorage.scala

```diff
@@ -165,21 +165,19 @@ object CylindricalThermalStorage {
       input: CylindricalStorageInput,
       initialStoredEnergy: Energy = zeroKWh,
   ): CylindricalThermalStorage = {
-
-    val maxEnergyThreshold: Energy =
-      CylindricalThermalStorage.volumeToEnergy(
-        CubicMeters(
-          input.getStorageVolumeLvl.to(Units.CUBIC_METRE).getValue.doubleValue
-        ),
-        KilowattHoursPerKelvinCubicMeters(
-          input.getC
-            .to(PowerSystemUnits.KILOWATTHOUR_PER_KELVIN_TIMES_CUBICMETRE)
-            .getValue
-            .doubleValue
-        ),
-        Celsius(input.getInletTemp.to(Units.CELSIUS).getValue.doubleValue()),
-        Celsius(input.getReturnTemp.to(Units.CELSIUS).getValue.doubleValue()),
-      )
+    val maxEnergyThreshold = volumeToEnergy(
+      CubicMeters(
+        input.getStorageVolumeLvl.to(Units.CUBIC_METRE).getValue.doubleValue
+      ),
+      KilowattHoursPerKelvinCubicMeters(
+        input.getC
+          .to(PowerSystemUnits.KILOWATTHOUR_PER_KELVIN_TIMES_CUBICMETRE)
+          .getValue
+          .doubleValue
+      ),
```

Comment:
In my opinion, it would be better to also move these two conversion to the utils class.

Reply:
To be honest, I won't do this since I guess we have this conversion only once at this place.

Reply:
Ok. My idea was, that it would look cleaner. But we can leave it as it is now.